build-tools/src/main/resources/checkstyle/checkstyle.xml

```diff
@@ -220,5 +220,11 @@
         <module name="CyclomaticComplexity"/>
         <module name="JavaNCSS" />
         <module name="NPathComplexity"/>
+
+        <module name="FinalParametersCheck">
```

Comment:
nit: Switch `FinalParamtersCheck` to `FinalParameters`, as none of the other checks postfix with `Check`